k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSetupBasics.java

```diff
@@ -277,8 +277,8 @@ private void finishAutoSetup() {
         String[] emailParts = splitEmail(email);
         String user = emailParts[0];
         String domain = emailParts[1];
-        URI incomingUri = null;
```

Comment:
It would be better to move the variable declaration down to where it's first declared. It's not used outside the block.